### PR TITLE
Master

### PR DESCRIPTION
**Pull request by rabbitu04**

### CODE REVIEW

File: mysite/crawler/crawler_for_windguru.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def get_data(data, begin):
 for i in range(len(locations)):
     html = requests.get(locations[i][2]).text
     sp = BeautifulSoup(html, 'html.parser')
-    data = sp.find_all('script', {'language': 'JavaScript'})[12].text
+    data = sp.find_all('script', {'language' : 'JavaScript'})[12].text
```
Comment (42B) [inline mysite/crawler/crawler_for_windguru.py]: Why would this space be added?

File: mysite/forecast/views.py [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
 
 # Create your views here.
 # coding : utf-8
 
-def show_data(request):
-    path = request.path
+def data_table(request, id):
+    if id:
+        path = request.path
+        location = Location.objects.get(id = id)
```
Comment (42B) [inline mysite/forecast/views.py]:
```python
.get(id=id)
```

**Comment by rabbitu04** [comment]
20181102

File: mysite/mysite/urls.py [hunk 3]
```diff
@@ -15,9 +15,11 @@
 """
 from django.contrib import admin
 from django.urls import path
-from forecast.views import show_data
+from forecast.views import *
```
Comment (42B) [inline mysite/mysite/urls.py]: > Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools.

* https://pep8.org/#imports

**Pull request merged**